presto-spark-base/src/test/java/com/facebook/presto/spark/execution/http/TestPrestoSparkHttpWorkerClient.java

```diff
@@ -275,13 +282,12 @@ public void testResultFetcherMultipleNonEmptyResults()
                 new TestingHttpClient(
                         new TestingResponseManager(
                             taskId.toString(),
-                                () -> NO_DURATION,
                             new TestingResponseManager.TestingResultResponseManager()
                             {
                                 private int requestCount;
 
                                 @Override
-                                public Response createResultResponse(String taskId) throws Exception
+                                public Response createResultResponse(String taskId) throws PageTransportErrorException
```

Comment:
move `throws PageTransportErrorException` to its own line. Same for all other throws